packages/fxa-content-server/app/scripts/views/would_you_like_to_sync.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import BackMixin from './mixins/back-mixin';
+import Cocktail from 'cocktail';
+import DoNotSyncMixin from './mixins/do-not-sync-mixin';
+import FlowEventsMixin from './mixins/flow-events-mixin';
+import FormView from './form';
+import Template from 'templates/would-you-like-to-sync.mustache';
+
+const SCREEN_CLASS = 'screen-would-you-like-to-sync';
+
+const View = FormView.extend(
```

Comment:
Is the account already created and email sent when this view is displayed? If so, does this view need to poll similar to CWTS in case the user verifies, or are we punting on that b/c of boomerang?

Reply:
boomerang + I think in most cases when there's a choice of SYNC vs NOT-Sync, we want to wait for the user...

@ryanfeeley might confirm